api/v1/installation_types.go

```diff
@@ -474,6 +474,17 @@ type CalicoNetworkSpec struct {
 	// Sysctl configures sysctl parameters for tuning plugin
 	// +optional
 	Sysctl []Sysctl `json:"sysctl,omitempty"`
+
+	// PolicySetupTimeoutSeconds delays new pods from going ready
+	// until their policy has been programmed in the dataplane.
+	// The specified delay defines the maximum amount of time
+	// that the Calico CNI will wait for policy to be programmed.
```

Comment:
"the Calico CNI _plugin" or just "Calico"

"CNI plugin" refers to a component, and "Calico" refers to the system as a whole.